pub-critic/src/app/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+import React, { useState } from "react";
+import chevron_down from "assets/chevron-down.svg";
+import chevron_up from "assets/chevron-up.svg";
+import classes from "./Switch.module.scss";
+import Image from "next/image";
+
+interface Props {
```

Comment:
`SwitchProps` would be better. So the idea is to call it `ComponentNameProps`. Here is okay, but sometimes you want to export that interface out and you can have many "Props"

Reply:
:white_check_mark: